### PR TITLE
Merge `mvp` to `develop` to use it as the default branch.

### PR DESCRIPTION
**Pull request by ochococo**
Make sure these boxes are checked before submitting your Pull Request - thank you!

- [x] [SwiftLint](https://github.com/Realm/SwiftLint) is installed on your system
- [ ] The project does not generate any warnings or errors
- [x] The tests are passing
- [x] Branch name is in line with GitFlow ([WTF?](http://jeffkreeftmeijer.com/2010/why-arent-you-using-git-flow/))

### CODE REVIEW

File: GrandCentralBoard/Widgets/Bonus/Source/PeopleWithBonususFetchController.swift [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+
+            switch result {
+            case .Success(let bonuses):
+                var allBonuses: [Bonus] = fetchedBonuses
+                allBonuses.appendContentsOf(bonuses)
+
+                let people = allBonuses.uniqueReceivers(strongSelf.preferredNumberOfPeople)
+
+                people.forEach { person in
+                    print(person.id)
```
Comment (karolkozub) [inline GrandCentralBoard/Widgets/Bonus/Source/PeopleWithBonususFetchController.swift]: This looks like something leftover after a debugging session.

**Comment by karolkozub** [comment]
SwiftLint gives four warnings: three regarding the use of `id` as a variable/property name, and one regarding function body length exceeding 40 characters

**Comment by karolkozub** [comment]
The local configuration.json should be changed not to include harvest. The current placeholder credentials crash the application.

File: GrandCentralBoard/App/Info.plist [hunk 2]
```diff
@@ -20,6 +20,12 @@
 	<string>????</string>
 	<key>CFBundleVersion</key>
 	<string>1</string>
+	<key>GCBAlwaysUseLocalConfigurationFile</key>
+	<false/>
+	<key>GCBLocalConfigurationFileName</key>
+	<string>configuration</string>
+	<key>GCBRemoteConfigurationPath</key>
+	<string>http://dr-doom.local/mateusz/gcb-config/raw/master/configuration.json</string>
```
Comment (karolkozub) [inline GrandCentralBoard/App/Info.plist]: I'm not sure if we want to include this URL in `develop`

File: GrandCentralBoard/Widgets/Bonus/Extensions/NSDate.swift [hunk 3]
```diff
@@ -8,22 +8,22 @@ import Foundation
 
 extension NSDate: Comparable { }
 
-public func ==(lhs: NSDate, rhs: NSDate) -> Bool {
+public func == (lhs: NSDate, rhs: NSDate) -> Bool {
```
Comment (karolkozub) [inline GrandCentralBoard/Widgets/Bonus/Extensions/NSDate.swift]: I thought we decided not to put a space before the opening paren for functions/methods. Is there an exception for operators?

**Comment by karolkozub** [comment]
I don't have any more comments. @ochococo

File: GrandCentralBoard/App/Info.plist (hunk 2 above)
Comment (ochococo) [inline GrandCentralBoard/App/Info.plist]: Changed to the one from GitHub repository.

File: GrandCentralBoard/Widgets/Bonus/Extensions/NSDate.swift (hunk 3 above)
Comment (ochococo) [inline GrandCentralBoard/Widgets/Bonus/Extensions/NSDate.swift]: I think this is a special case.

**Pull request merged**